react/week2/todo-list/src/TodoItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './TodoItem.css';
+
+function TodoItem(props){
+    const {description, id, deleteTodo, todoItemClicked} = props;
+    const { isChecked } = props;
+
+    const todoItemChecked = event => {
```

Comment:
I think it's more clear to write your changeHandlers as something that will happen, as opposed to something that has happened: 
```suggestion
    const checkTodoItem = event => {
```